### ISSUE TITLE
Tree status is not refreshed on old PRs

### ISSUE THREAD

**Issue by keyonghan** [flutter/flutter#106816]
For example: https://github.com/flutter/flutter/pull/101834

Tree has been green since yesterday, but the `luci-flutter` status is still red.
<img width="1033" alt="Screen Shot 2022-06-29 at 9 50 26 AM" src="https://user-images.githubusercontent.com/54558023/176492299-31e8c11c-8b71-43bd-a0a1-0427d57d301b.png">

Culprit: [pull list api ](https://docs.github.com/en/rest/pulls/pulls#list-pull-requests)returns 30 PRs by default. We should extend the number to cover relatively old PRs.

**Comment by jmagman** [comment]
Does this mean only 30 of our 199 open PRs are getting luci-flutter updates?

**Comment by keyonghan** [comment]
> Does this mean only 30 of our 199 open PRs are getting luci-flutter updates?

Right. The top 30 PRs based on `created`. To avoid creating new PRs, one workaround to to switch to `updated`. I guess we switched to a smaller number instead of all PRs due to API quote issue.
